core/controllers/AlbumController.js

```diff
@@ -0,0 +1,16 @@
+import Album from '../models/Album.js';
+
+class AlbumController {
+  async getAlbumsByAuthor(req, res, next) {}
+  async getAlbumsPublicByAuthor(req, res, next) {}
+  async getLikesById(req, res, next) {}
```

Comment:
The AlbumSchema already has `like` attr, should we have this route?

Reply:
router.get('/:user-id/', photoController.getPhotosByAuthor); Dùng để lấy danh sách album (private và public) của 1 user bất kỳ theo id.
router.get('/public/:user-id/', photoController.getPhotosPublicByAuthor); Dùng để lấy danh sách album (chỉ public) của 1 user bất kỳ theo id.
router.get('/like/:photo-id/', photoController.getLikesById); Dùng để hiển thị số like của 1 album(icon trái tym).